lib/ui/blocs/navigator/navigator_bloc.dart

```diff
@@ -77,7 +79,8 @@ class NavigatorBloc extends Bloc<NavigatorEvent, void> {
   }
 
   void _mapNavigateToHomeEventToState() {
-    _navigatorKey.currentState.pushAndRemoveUntil(
+    dynamic pushMethod = _navigatorKey.currentState.pushAndRemoveUntil;
+    pushMethod(
```

Comment:
Why did you declare the function as a variable? I think it makes the code harder to comprehend and it's better to switch to as it was before

```suggestion
    _navigatorKey.currentState.pushAndRemoveUntil(
```

Reply:
Forgot to clean up. Originally i had something like this going on
```
dynamic pushMethod = a;
if (statement) {
  dynamic pushMethod = b;
}

pushMethod(...push);
```